emails/utils.py

```diff
@@ -375,4 +390,29 @@ def count_all_trackers(html_content):
     study_logger.info(
         'email_tracker_summary',
         extra={'general': general_detail, 'strict': strict_detail}
-    )
\ No newline at end of file
+    )
+
+def convert_domains_to_regex_patterns(domain_pattern):
+    return '(["])(\\S*://\\S*.' + domain_pattern + '\\S*)\\1'
+
+def remove_trackers(html_content, level='general'):
+    trackers = GENERAL_TRACKERS if level == 'general' else STRICT_TRACKERS
+    tracker_count = 0
+    changed_content = html_content
+
+    general_detail = count_tracker(html_content, GENERAL_TRACKERS)
+    strict_detail = count_tracker(html_content, STRICT_TRACKERS)
+    
+    for tracker in trackers:
+        pattern = convert_domains_to_regex_patterns(tracker)
+        changed_content, matched = re.subn(pattern, f'\g<1>{settings.SITE_ORIGIN}/faq\g<1>', changed_content)
+        tracker_count += matched
+
+    
+    incr_if_enabled('tracker_foxfooding.general_count', tracker_count)
+    incr_if_enabled('tracker_foxfooding.strict_count', strict_detail['count'])
+    study_logger.info(
+        'email_tracker_foxfooding_summary',
+        extra={'general': general_detail, 'strict': strict_detail}
```

Comment:
Can we add `tracker_count` and `level` to the log as well?

Reply:
done by 916082ab643b88179ed5eccba89d6aab82f411c7